grails-app/conf/UrlMappings.groovy

```diff
@@ -60,6 +60,11 @@ class UrlMappings {
         }
 
 
+        "/api/loadData"(parseRequest: true) {
```

Comment:
Let's make this more REST-like And we need to do some thinking about naming convention here. 

Something like this maybe
```
GET /api/config/data/demo
```
But I'll do some more thinking about this tomorrow.

Reply:
For now changed to /api/config/data/demo